flutter_appauth/example/lib/main.dart

```diff
@@ -209,7 +209,12 @@ class _MyAppState extends State<MyApp> {
   Future<void> _signInWithNoCodeExchange() async {
     try {
       _setBusyState();
-      // use the discovery endpoint to find the configuration
+      /* 
+         The discovery endpoint (_discoveryUrl) is used to find the configuration.
+         The code challenge generation can be checked here > https://github.com/MaikuB/flutter_appauth/search?q=challenge.
+         The code challenge is generated from the code verifier and only the code verifier is included in the result.
+         This because to get the token in the method _exchangeCode (see above) we need only the code verifier and the authorization code according to the spec https://www.rfc-editor.org/rfc/rfc7636 page 9 section 4.5.
```

Comment:
Thanks for making the change. I opened this an IDE and saw this column was fairly long that I needed to horizontally scroll to see the rest of the comments. Are you able to make each line of comment follow the line length of at most 80 characters (i.e. make it consistent the default for Flutter code)?